sdk/test-utils/perfstress/GettingStarted.md

```diff
@@ -261,3 +262,61 @@ Example: Currently `@azure/<service-sdk>` is at 12.4.0 on master and you want to
 - Navigate to `sdk\storage\perf-tests\<service-sdk>`
 - `rush build -t perf-test-<service-sdk>`
 - Run the tests as suggested before, example `npm run perf-test:node -- TestClassName --warmup 2 --duration 7 --iterations 2 --parallel 50`
+
+## [Using Proxy Tool](#using-proxy-tool)
+
+### Using the testProxy option
+
+To be able to leverage the powers of playing the back the requests using the test proxy, add the following to your code.
+
+      ```ts
+      /// Core V1 SDKs - For services depending on core-http
+      /// Pass the proxy client as the httpClient in the client options
+      this.blobServiceClient = BlobServiceClient.fromConnectionString(
+        connectionString,
+        {
+          httpClient: this.getHttpClientV1()
+        }
+      );
+
+      /// Core V2 SDKs - For services depending on core-rest-pipeline
+      /// Use the addPolicy call to add the test proxy policy
+      this.client = TableClient.fromConnectionString(connectionString, tableName);
+      this.client.pipeline.addPolicy(testProxyHttpPolicy(this.getHttpClient()));
+
+      // Not all core-v1 SDKs allow passing httpClient option.
+      // Not all core-v2 SDKs allow adding policies.
+      // Please reach out if your service doesn't support.
+      ```
+
+### Running the proxy server
+
+Run this command
+
+- `docker run -p 5000:5000 azsdkengsys.azurecr.io/engsys/ubuntu_testproxy_server:latest`
+
+If the above command doesn't work directly, try logging in. [@Scott Beddall is trying to iron out the kinks :)]
```

Comment:
Nobody should hit this anymore! Only those who had to pull down from it when it WAS secured will hit this issue anymore. [Reference](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/docker/README.md#troubleshooting-access-to-public-container-registry)